### PR TITLE
updated pagerduty

### PR DESCRIPTION
**Pull request by jerry-li-dev**
Updated screenshots and steps

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/pagerduty.md [hunk 1]
```diff
@@ -25,43 +25,39 @@ You can use this integration to trigger incidents in PagerDuty based on your Pos
...
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-search-all.jpg)
 
-1. Click the **View Details** button to see information about PagerDuty.
+You can select `View All` to see all PagerDuty integrations created by your team.
 
-   [![select pagerduty integration](https://assets.postman.com/postman-docs/WS-integrations-pagerduty-details.png)](https://assets.postman.com/postman-docs/WS-integrations-pagerduty-details.png)
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-view-all.jpg)
 
-   You can also click the **Configured Integrations** tab to set up other integrations, view available integrations for PagerDuty, or view all integrations.
+Select `Add Integration` to create your own integration
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:52]: I'd suggest rephrasing this to "Click `Add Integration` to create your own integration."

File: src/pages/docs/integrations/available-integrations/pagerduty.md [hunk 2]
```diff
@@ -25,43 +25,39 @@ You can use this integration to trigger incidents in PagerDuty based on your Pos
...
+From the [Integrations search page](https://go.postman.co/integrations/browse?category=all), search and select PagerDuty.
 
-[![select pagerduty integration](https://assets.postman.com/postman-docs/integrations-pagerduty1.png)](https://assets.postman.com/postman-docs/integrations-pagerduty1.png)
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-search-all.jpg)
 
-1. Click the **View Details** button to see information about PagerDuty.
+You can select `View All` to see all PagerDuty integrations created by your team.
 
-   [![select pagerduty integration](https://assets.postman.com/postman-docs/WS-integrations-pagerduty-details.png)](https://assets.postman.com/postman-docs/WS-integrations-pagerduty-details.png)
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-view-all.jpg)
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:50]: Image repetition here - let's just keep one.

File: src/pages/docs/integrations/available-integrations/pagerduty.md [hunk 3]
```diff
@@ -25,43 +25,39 @@ You can use this integration to trigger incidents in PagerDuty based on your Pos
...
 ## Configuring Postman with PagerDuty
 
-1. In the [Integrations](https://go.postman.co/workspaces) tab for your workspace, select PagerDuty from the list of third party integrations.
+From the [Integrations search page](https://go.postman.co/integrations/browse?category=all), search and select PagerDuty.
 
-[![select pagerduty integration](https://assets.postman.com/postman-docs/integrations-pagerduty1.png)](https://assets.postman.com/postman-docs/integrations-pagerduty1.png)
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-search-all.jpg)
 
-1. Click the **View Details** button to see information about PagerDuty.
+You can select `View All` to see all PagerDuty integrations created by your team.
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:48]: You can combine this to - "You can select `View All` to see all PagerDuty integrations created by your team. To create your own integration, click `Add Integration`.

File: src/pages/docs/integrations/available-integrations/pagerduty.md (hunk 1 above)
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:52]: Added above, you can remove this line.

File: src/pages/docs/integrations/available-integrations/pagerduty.md [hunk 4]
```diff
@@ -25,43 +25,39 @@ You can use this integration to trigger incidents in PagerDuty based on your Pos
...
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-view-all.jpg)
 
-   You can also click the **Configured Integrations** tab to set up other integrations, view available integrations for PagerDuty, or view all integrations.
+Select `Add Integration` to create your own integration
 
-## Add a team activity feed to PagerDuty
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-add-integration.jpg)
 
-The Team Activity Feed is where you can track changes being made by your team members to shared Collections. With this integration you can stay updated about work being done in Postman while you’re away from the desktop app, either through your PagerDuty dashboard or through an alternative alert service you choose to set up.
+Enter the name of the integration, workspace, monitor, and your PagerDuty Integration Key. Then, select `Add Integration`.
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:56]: It's "Pagerduty Integration Key" in the screenshot. Can you check on this? Its PagerDuty everywhere else.

File: src/pages/docs/integrations/available-integrations/pagerduty.md [hunk 5]
```diff
@@ -25,43 +25,39 @@ You can use this integration to trigger incidents in PagerDuty based on your Pos
...
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-add-integration-configuration.jpg)
 
-1. In the PagerDuty integration page, click the **Add Integration** button.
-1. In the **Team Activity Feed** page, choose the Monitor you want to send to PagerDuty.
-1. Enter your Pagerduty Integration Key and click the **Add Integration** button.
-
-[![pagerduty monrun](https://assets.postman.com/postman-docs/WS-integrations-pagerduty-monrun.png)](https://assets.postman.com/postman-docs/WS-integrations-pagerduty-monrun.png)
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-all-integrations.jpg)
 
 ## Viewing PagerDuty results
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:62]: Change image format to .jpg

File: src/pages/docs/integrations/available-integrations/pagerduty.md [hunk 6]
```diff
@@ -25,43 +25,39 @@ You can use this integration to trigger incidents in PagerDuty based on your Pos
...
+Select `Add Integration` to create your own integration
 
-## Add a team activity feed to PagerDuty
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-add-integration.jpg)
 
-The Team Activity Feed is where you can track changes being made by your team members to shared Collections. With this integration you can stay updated about work being done in Postman while you’re away from the desktop app, either through your PagerDuty dashboard or through an alternative alert service you choose to set up.
+Enter the name of the integration, workspace, monitor, and your PagerDuty Integration Key. Then, select `Add Integration`.
 
-To add a team activity feed to PagerDuty:
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-add-integration-configuration.jpg)
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:59]: If you could add a line, what happens after clicking on `Add Integration`

File: src/pages/docs/integrations/available-integrations/pagerduty.md [hunk 7]
```diff
@@ -25,43 +25,39 @@ You can use this integration to trigger incidents in PagerDuty based on your Pos
...
 
-1. In the PagerDuty integration page, click the **Add Integration** button.
-1. In the **Team Activity Feed** page, choose the Monitor you want to send to PagerDuty.
-1. Enter your Pagerduty Integration Key and click the **Add Integration** button.
-
-[![pagerduty monrun](https://assets.postman.com/postman-docs/WS-integrations-pagerduty-monrun.png)](https://assets.postman.com/postman-docs/WS-integrations-pagerduty-monrun.png)
+![pagerduty menu](https://assets.postman.com/postman-docs/pagerduty-all-integrations.jpg)
 
 ## Viewing PagerDuty results
```
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:63]: Change the last image format to .jpg

File: src/pages/docs/integrations/available-integrations/pagerduty.md (hunk 4 above)
Comment (sowmyaraj92) [inline src/pages/docs/integrations/available-integrations/pagerduty.md:56]: I would rephrase this " Then, select `Add Integration` " to " Click `Add Integration`."

**Review by sowmyaraj92 — CHANGES_REQUESTED** [review]
@jerry-li-dev A few changes to be made and we are good to go!

File: src/pages/docs/integrations/available-integrations/pagerduty.md (hunk 4 above)
Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/pagerduty.md:56]: it should be PagerDuty. It is an UX issue.

Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/pagerduty.md:56]: @jotsarup

**Review by jkonrath-postman — APPROVED** [review]
LGTM
